{"uuid": "4a1366d3-a6d7-44a1-8ead-9fe10234d240", "children": ["e9b9eb19-3484-4373-9b72-596f3e4e9ad2"], "befores": [{"name": "setup", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 300.000\n  (Session info: chrome=125.0.6422.76)\nStacktrace:\n\tGetHandleVerifier [0x00007FF6F78A1F52+60322]\n\t(No symbol) [0x00007FF6F781CEC9]\n\t(No symbol) [0x00007FF6F76D7EBA]\n\t(No symbol) [0x00007FF6F76C070D]\n\t(No symbol) [0x00007FF6F76C0410]\n\t(No symbol) [0x00007FF6F76BE39D]\n\t(No symbol) [0x00007FF6F76BEBAF]\n\t(No symbol) [0x00007FF6F76CD97E]\n\t(No symbol) [0x00007FF6F76E5C6F]\n\t(No symbol) [0x00007FF6F76EB67A]\n\t(No symbol) [0x00007FF6F76BF33A]\n\t(No symbol) [0x00007FF6F76E59B1]\n\t(No symbol) [0x00007FF6F776BB5B]\n\t(No symbol) [0x00007FF6F774BFC3]\n\t(No symbol) [0x00007FF6F7719617]\n\t(No symbol) [0x00007FF6F771A211]\n\tGetHandleVerifier [0x00007FF6F7BB949D+3301613]\n\tGetHandleVerifier [0x00007FF6F7C036C3+3605267]\n\tGetHandleVerifier [0x00007FF6F7BF9440+3563664]\n\tGetHandleVerifier [0x00007FF6F7954326+790390]\n\t(No symbol) [0x00007FF6F782750F]\n\t(No symbol) [0x00007FF6F7823404]\n\t(No symbol) [0x00007FF6F7823592]\n\t(No symbol) [0x00007FF6F7812F9F]\n\tBaseThreadInitThunk [0x00007FFA56C5257D+29]\n\tRtlUserThreadStart [0x00007FFA5736AA48+40]\n\n", "trace": "  File \"C:\\Users\\User\\PycharmProjects\\pythonProjectseleniumframework\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Users\\User\\PycharmProjects\\pythonProjectseleniumframework\\venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\pythonProjectseleniumframework\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\pythonProjectseleniumframework\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1122, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\pythonProjectseleniumframework\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 877, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\PycharmProjects\\pythonProjectseleniumframework\\tests\\conftest.py\", line 18, in setup\n    driver.maximize_window()\n  File \"C:\\Users\\User\\PycharmProjects\\pythonProjectseleniumframework\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 499, in maximize_window\n    self.execute(command, None)\n  File \"C:\\Users\\User\\PycharmProjects\\pythonProjectseleniumframework\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 347, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\User\\PycharmProjects\\pythonProjectseleniumframework\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1716357558467, "stop": 1716357980547}], "start": 1716357558467, "stop": 1716357980755}